### PR TITLE
Fix tests for Python 3.7

### PR DESCRIPTION
**Pull request by sbraz**
See commit messages for more details.

### CODE REVIEW

File: tests/test_future/test_int.py [hunk 1]
```diff
@@ -265,6 +265,9 @@ def test_small_ints(self):
     def test_no_args(self):
         self.assertEqual(int(), 0)
 
+    @unittest.skipIf(sys.version_info >= (3, 7),
```
Comment (serhiy-storchaka) [inline tests/test_future/test_int.py]: The test for `int('100', base=2)` shouldn't be skipped.

Comment (sbraz) [reply tests/test_future/test_int.py]: Yeah, this one still works but I didn't think it was worth keeping. Would you like me to split the test in two functions.

Comment (jmadler) [reply tests/test_future/test_int.py]: Yes please! :)

**Comment by sbraz** [comment]
@jmadler repushed

**Pull request merged**